data/json/flags.json

```diff
@@ -85,6 +85,11 @@
     "type": "json_flag",
     "info": "This item can be used to block attacks when worn."
   },
+  {
+    "id": "CALORIE_INTAKE",
```

Comment:
Just a nitpick, but corresponding iuse action has `CALORIES` and flag has just `CALORIE` in it.

Reply:
thanks, good catch.